src/npms-package-info.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const moment = require('moment');
+const got = require('got');
+const log = require('./logger');
+
+module.exports = list => {
+  log.info('npmsInfo: Fetching info for %s packages', list.length);
+  let arrayPackages = [];
```

Comment:
this seems to do a lot more work than 

```
return Promise.all(list.map(pkg => createPromiseUpdatePage(pkg)
.then(pkgs => {		
  pkgs = pkgs.filter(pkg => Boolean(pkg));		
  log.info('npmInfo: Fetched info for %s valid packages', pkgs.length);		
  return pkgs;		
});
```

and return the `formattedPkg` in `createPromiseUpdatePage`